### DISCUSSION TITLE
Adding a config entry to use a specific git binary

### DISCUSSION THREAD

**Discussion by ghost** [jesseduffield/lazygit#3138] [Ideas]
Can be useful if working in an environment where the installed git is an older version, using a new config entry would allow us to compile our version of git that is compatible and use it.

**Comment by stefanhaller** [comment]
This doesn't need a config entry. Lazygit will use the first git that it finds in PATH, so just set your PATH so that your own compiled version comes before the system one. E.g. `export PATH=$HOME/bin:$PATH`, assuming you compiled git into `$HOME/bin`.
